docs/architecture/microservices/multi-container-microservice-net-applications/subscribe-events.md

```diff
@@ -366,9 +366,6 @@ If the "redelivered" flag is set, the receiver must take that into account, beca
 - **Eric Brewer. CAP Twelve Years Later: How the "Rules" Have Changed** \
     <https://www.infoq.com/articles/cap-twelve-years-later-how-the-rules-have-changed>
 
-- **Azure Service Bus. Brokered Messaging: Duplicate Detection**  \
-    <https://code.msdn.microsoft.com/Brokered-Messaging-c0acea25>
-
```

Comment:
In case you prefer to re-port the sample later, the original samples should be in https://github.com/microsoftarchive?q=msdn-code&type=&language=&sort=

Reply:
Thanks @Youssef1313 !